pages.es/linux/showkey.md

```diff
@@ -0,0 +1,24 @@
+# showkey
+
+> Muestra el código de las teclas pulsadas en el teclado, útil para depurar problemas relacionados con el teclado y la reasignación de teclas.
+> Más información: <https://manned.org/showkey>.
+
+- Visualiza códigos de teclas en decimal:
+
+`sudo showkey`
+
+- Visualiza [s]cancodes en hexadecimal:
```

Comment:
```suggestion
- Visualiza códigos de ra[s]treo en hexadecimal:
```

I think according to an [Ubuntu man page](https://manpages.ubuntu.com/manpages/trusty/es/man1/showkey.1.html) the word scancode translates to `código de rastreo`. I did some digging and [apparently it is a concept](http://galia.fc.uaslp.mx/~cantocar/ARQUI_COM_II/LIBRO%20IBM-PC/ap05.htm)